cmd/swarm-rafttool/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	mainCmd = &cobra.Command{
+		Use:   os.Args[0],
+		Short: "Tool to translate and decrypt the raft logs of a swarm manager",
+	}
+
+	decryptCmd = &cobra.Command{
+		Use:   "decrypt <output directory>",
+		Short: "Decrypt a swarm manager's raft logs to an optional directory",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return errors.New("output directory required")
+			}
```

Comment:
Maybe the output directory should be given with a flag, that has some default value. That would be more consistent with the input directory.